content/post/098-libp2p-in-2020.md

```diff
@@ -0,0 +1,115 @@
+
+What is Testground you ask? Only the coolest new testing tool for large-scale p2p networks! When starting our network upgrade plans, it was clear that the scope and complexity of changes we’d need to make to core IPFS network functions would be impossible to optimize without powerful testing tools to simulate each change to network performance at scale. We couldn’t find a tool with the scale and configurability we needed - so we built the testing tool we needed, and made it modular and system agnostic just like libp2p so other p2p networks can take advantage of this new OSS infrastructure too!
+
+### Blockchains
+
+For our large blockchain users, the focus of Q1 was stability, scalability and security hardening. We designed and implemented a new version for the Gossipsub protocol, v1.1. This new P2P PubSub router includes several security extensions that add a protection to Sybil, eclipse and spam attacks. This work is of extreme importance as our adopters (e.g. Filecoin and Ethereum 2.0) require a secure messaging layer to distribute their time sensitive and valuable information, without being vulnerable to malicious actors. You can find the [specification](https://github.com/libp2p/specs/tree/master/pubsub/gossipsub) and [implementation](https://github.com/libp2p/go-libp2p-pubsub/) on Github. We will be releasing a report that fully describes how we identified the attack vectors, developed the motivations and went through on implementing, testing, evaluating under multiple network conditions and got it audited. Stay tuned.
+
+Both of these workstreams (content routing performance and gossipsub hardening) continue to be top priorities  with IPFS having launched go-ipfs 0.5, their largest network upgrade in years, and Ethereum and Filecoin gearing up for Q3 mainnet launches. 
+
+## Growing the team
+
+Protocol Labs’s small and scrappy libp2p core team transformed into a true force of nature with some great additions last year. Our doors will continue to be open for great people from both inside and outside of the ecosystem. 
+
+In particular, **we’re looking for a new Project Lead** to help grow and steward the libp2p project and ecosystem to new heights in 2020. Check out the [job description](https://hire.lever.co/jobs/internal/27ff3891-6e13-4aa8-b43a-734715e85a26) where you can apply. In particular, we’re looking for candidates with proven leadership experience, a passion for p2p tech, and an interest in scaling the libp2p community as it becomes critical infrastructure for new, large users such as Ethereum (with launch of the Beacon chain) and Filecoin.
```

Comment:
This lever link needs to be published or something. You can’t view it unless you login with a protocol.ai address.